rust-runtime/aws-smithy-runtime-api/src/client/runtime_plugin.rs

```diff
@@ -99,6 +99,41 @@ impl RuntimePlugins {
     }
 }
 
+#[derive(Default, Debug)]
+pub struct PassthroughRuntimePlugin {
+    config: Option<FrozenLayer>,
+    runtime_components: Option<RuntimeComponentsBuilder>,
+}
```

Comment:
`RuntimePluginBuilder` maybe? `StaticRuntimePlugin`

Reply:
Renamed in cacf7bac9993f8785ba087aa185e4fb69d5c1419